index.html

```diff
@@ -4,9 +4,53 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+    <link rel="stylesheet" type="text/css" href="styles/index.css">
+    <title>Sabrina's Weather Report</title>
 </head>
 <body>
-    
+    <header class="header_class">
```

Comment:
found a few errors when I sent this through the w3 html validator. feel free to check it out here: [https://validator.w3.org/](https://validator.w3.org)